src/pages/ChallengeDetails/ChallengeDetails.jsx

```diff
@@ -0,0 +1,158 @@
+            "Provide constructive feedback to your partner's code and suggest improvements"
+        ],
+        steps: [
+            ["Build the card component", ["Use the HTML to structure the card, with an image, title, description, and hidden section for additional text."]],
+            ["Submit your code", ["After building your component, input your code into the designated User Code section and click Submit."]],
+            ["Review you partners code", ["After submitting your code, your partner’s code will appear in the Partner’s Code section. Review their component and provide feedback on areas for improvement."]],
+            ["Finalize component", ["Discuss and apply any feedback with your partner to refine your components. Make sure everything works smoothly", "Click Complete once both of you are satisfied with the final version."]]
+        ]
+
+
+    }
+
+    return (
+        <>
+            <DashboardNavbar />
+            <main className="main">
+
+                <img src={placeholderData.imgSrc} className="image" alt="computer on a desk" />
+
+                <section className="challenge">
+
+                    <div className="overview">
+                        <div className="overview__difficulty">
+                            {challengeData.difficulty}
+                        </div>
+                        <h1 className="overview__title">{challengeData.task}</h1>
+                        <p className="overview__overview">{challengeData.detailedProblem.description}</p>
+
+                        <div className="overview__details">
+                            <div className="overview__type">
+                                <img src={pen} className="overview__icon" alt="pen icon" />
+                                <span className="overview__type-text">{challengeData.category}</span>
+                            </div>
+                            <div className="overview__type">
+                                <img src={clock} className="overview__icon" alt="pen icon" />
+                                <span className="overview__type-text">{challengeData.duration}</span>
+                            </div>
+                            <div className="overview__type">
+                                <img src={bulb} className="overview__icon" alt="pen icon" />
+                                <span className="overview__type-text">VSCode / Figma</span>
+                            </div>
+                        </div>
+
+                        <div className="objectives">
+                            <h2 className="objectives__title">Constraints</h2>
+                            <ul className="objectives__list">
+                                {challengeData.detailedProblem.constraints.map(constraint => {
+                                    return (
+                                        <li className="objectives__item" key={constraint}>{constraint}</li>
```

Comment:
constraint should not be used as a key. at worst use index as the key

Reply:
I changed the key to the index. Would it be ideal to generate a UID for each constraint?